grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy

```diff
@@ -157,6 +157,10 @@ class InvoiceService {
         return invoiceItemCandidates
     }
 
+    def saveInvoice(Invoice invoice) {
+        invoice.save(flush: true, failOnError: true)
```

Comment:
I know it was existing like that, but we've had a discussion not a long time ago, that we should avoid throwing `flush: true` everywhere. It necessary here?

Reply:
Looks like it was not required.